ci/terraform/auth-external-api/auth-ext-api-gateway.tf

```diff
@@ -6,18 +6,14 @@ locals {
 data "aws_vpc" "auth_shared_vpc" {
   filter {
     name   = "tag:Name"
-    values = ["${var.environment}-shared-vpc"]
+    values = [var.environment == "authdev2" ? "dev-shared-vpc" : "${var.environment}-shared-vpc"]
   }
 
 }
 
 data "aws_vpc_endpoint" "auth_api_vpc_endpoint" {
   vpc_id       = data.aws_vpc.auth_shared_vpc.id
   service_name = "com.amazonaws.eu-west-2.execute-api"
-  tags = {
```

Comment:
merge issue with tags?

Reply:
No , These tags were applied to data read  not resource , were causing problem in getting the vpc endpoint ID of dev vpc , this should be fine as we need cost tags on resource

Reply:
see my comment about locals / `var.vpc_environment`